.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    extends: ['airbnb', 'prettier', 'prettier/flowtype', 'prettier/react'],
```

Comment:
Because airbnb has also a lot of rules regarding flow and react that people seem to like and I would prefer not to waste a lot of time trying to come up with configs for those (and a few more e.g. a11y)